website/content/en/docs/concepts/provisioners.md

```diff
@@ -70,7 +70,7 @@ spec:
     - key: "karpenter.k8s.aws/instance-hypervisor"
       operator: In
       values: ["nitro"]
-    - key: karpenter.k8s.aws/instance-generation
```

Comment:
Can you add this into the `/preview` path to the docs as well? This is the path that is pulled forward from version to version?